### PR TITLE
Fix for 'Show Solutions'. Correct format is now displayed.

### PR DESCRIPTION
**Pull request by NVieregg**
Drag&Drop should now display Solutions in correct format. Please check if

- [x] Format is now correct for Check Solution. Only Category and Answer should be displayed.
- [x] Check if Simple Language also works the same.

Note: we should consider renaming Categories and Items. Discuss in tomorrow's meeting.

### CODE REVIEW

**Review by cmauf — COMMENTED** [review]
We might consider adding a warning to course authors when they did not specify a correct answer. Also applies to #305

**Comment by NVieregg** [comment]
> We might consider adding a warning to course authors when they did not specify a correct answer. Also applies to #305

As we agreed on pushing the warnings into the big refactoring, please check this again, if everything is correct now and then merge. Thanks!

File: src/plugins/laya-la-drag-drop/view.vue [hunk 1]
```diff
@@ -127,7 +127,7 @@ Dependencies:
         v-for="(item, index) in items"
         :key="index"
       >
-        {{ item.label }}: {{ categories[items[index].category] }},
+        {{ item.label }}: {{ courseSimple? categories[items[index].category].simple: categories[items[index].category].text }}
```
Comment (cmauf) [inline src/plugins/laya-la-drag-drop/view.vue:130]: simplify to `courseSimple? item.simple: item.text` for better readability

**Review by cmauf — CHANGES_REQUESTED** [review]
improve readability

**Review by LAYAIraq — APPROVED** [review]

**Review by cmauf — APPROVED** [review]

**Pull request merged**